packages/docusaurus/src/webpack/getProcessForPort.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import chalk from 'chalk';
+import {execSync} from 'child_process';
+import path from 'path';
+
+const execOptions: object = {
+  encoding: 'utf8',
+  stdio: [
+    'pipe', // stdin (default)
+    'pipe', // stdout (default)
+    'ignore', // stderr
+  ],
+};
+
+// Checks if process is a React app
+function isProcessAReactApp(processCommand) {
+  return /^node .*react-scripts\/scripts\/start\.js\s?$/.test(processCommand);
+}
+
+// Gets process id of what is on port
+function getProcessIdOnPort(port: number): string {
+  return execSync(`lsof -i:${port} -P -t -sTCP:LISTEN`, execOptions)
+    .toString()
+    .split('\n')[0]
+    .trim();
+}
+
+// Gets package name in a directory
+async function getPackageNameInDirectory(
```

Comment:
do need to export it afaik

Reply:
Are you saying to remove the `export default function getProcessForPort` ? I am using the function in another file.